### PR TITLE
add desktop menu entry

### PR DESCRIPTION
**Pull request by be1**

### CODE REVIEW

File: data/tascamgtk.desktop [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[Desktop Entry]
+Encoding=UTF-8
+Name=TascamGtk
+Name[fr]=TascamGtk
```
Comment (trebmuh) [inline data/tascamgtk.desktop]: @be1 : if it not translated, it's useless to add this line.

File: data/tascamgtk.desktop [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+GenericName=Tascam US-16x08 DSP mixer application
+GenericName[fr]=contrôle du DSP de la Tascam US-16x08
```
Comment (trebmuh) [inline data/tascamgtk.desktop]: In most (if not all) other desktop files, the GenericName start with a capital letter.

File: data/tascamgtk.desktop [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+Comment=controls the hardware mixer of the Tascam US-16x08 sound card
+Comment[fr]=contrôle le mixeur matériel de la carte son Tascam US-16x08
```
Comment (trebmuh) [inline data/tascamgtk.desktop]: same (capital letter)

**Comment by trebmuh** [comment]
👍

**Comment by be1** [comment]
Thank you for your comments. Last commit should fix the mentioned typos.

**Comment by trebmuh** [comment]
You're welcome.
Thank you for this job.

**Pull request merged**